File: apiFilRougeIb/Models/UserAnswer.cs
namespace apiFilRougeIb.Models
{
    public class UserAnswer
    {
        public long User_IdUser { get; set; }
        public long Answer_IdAnswer { get; set; }
        public long Question_IdQuestion { get; set; }

        public UserAnswer() { }

        public UserAnswer(long idUser, long answer_IdAnswer, long question_IdQuestion)
        {
            User_IdUser = idUser;
            Answer_IdAnswer = answer_IdAnswer;
            Question_IdQuestion = question_IdQuestion;
        }
        public override string ToString()
        {
            return $"{this.User_IdUser} {this.Answer_IdAnswer} {this.Question_IdQuestion}";
        }
    }
}

File: angularFilRougeIb/src/app/_models/userAnswer.ts
import { time } from "console";
import { tmpdir } from "os";

export class UserAnswer{

  private _user_IdUser : number;
  private _answer_IdAnswer : number;
  private _question_IdQuestion : number;

  constructor(user_IdUser : number, answer_IdAnswer : number, question_IdQuestion : number){
    this._user_IdUser = user_IdUser;
    this._answer_IdAnswer = answer_IdAnswer;
    this._question_IdQuestion = question_IdQuestion;
  }

  get user_IdUser() : number {
    return this._user_IdUser;
  }

  get answer_IdAnswer() : number {
    return this._answer_IdAnswer;
  }

  get question_IdQuestion() : number {
    return this._question_IdQuestion;
  }

}
